examples/with-actix/prisma/schema.prisma

```diff
@@ -0,0 +1,24 @@
+datasource db {
+  provider             = "mysql"
```

Comment:
I'd prefer that examples all use sqlite, since that can be used without starting another database